core/src/banking_stage/committer.rs

```diff
@@ -134,7 +134,13 @@ impl Committer {
         starting_transaction_index: Option<usize>,
     ) {
         if let Some(transaction_status_sender) = &self.transaction_status_sender {
-            let txs = batch.sanitized_transactions().to_vec();
+            // Clone `SanitizedTransaction` out of `RuntimeTransaction`, this is
+            // done to send over the status sender.
+            let txs = batch
+                .sanitized_transactions()
+                .iter()
+                .map(|tx| tx.deref().clone())
```

Comment:
will be replaced by use of upcoming trait `SVMTransactionAdapter` in a future PR